service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/diagnostic_report/DiagnosticReportMapper.java

```diff
@@ -0,0 +1,78 @@
+package uk.nhs.adaptors.gp2gp.ehr.mapper.diagnostic_report;
+
+import com.github.mustachejava.Mustache;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+
+import org.apache.commons.lang3.StringUtils;
+import org.hl7.fhir.dstu3.model.DiagnosticReport;
+import org.hl7.fhir.dstu3.model.Observation;
+import org.hl7.fhir.dstu3.model.ResourceType;
+import org.hl7.fhir.dstu3.model.Specimen;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import uk.nhs.adaptors.gp2gp.ehr.mapper.IdMapper;
+import uk.nhs.adaptors.gp2gp.ehr.mapper.MessageContext;
+import uk.nhs.adaptors.gp2gp.ehr.mapper.parameters.diagnostic_report.DiagnosticReportCompoundStatementTemplateParameters;
+import uk.nhs.adaptors.gp2gp.ehr.utils.DateFormatUtil;
+import uk.nhs.adaptors.gp2gp.ehr.utils.TemplateUtils;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+@Component
+@Slf4j
+public class DiagnosticReportMapper {
```

Comment:
Is this supposed to be in here, as it exists on @matthewdoreilly's branch?

Reply:
it could be to do with the package name actually. I believe the underscore was removed from `diagnostic_report`

Reply:
Yes, just notices that directories names has changed, new commit is fixing it.

Reply:
Thanks